src/material/datepicker/datepicker.md

```diff
@@ -8,29 +8,17 @@ the calendar. It is made up of several components and directives that work toget
 A datepicker is composed of a text input and a calendar pop-up, connected via the `matDatepicker`
 property on the text input.
 
-```html
-<input [matDatepicker]="myDatepicker">
-<mat-datepicker #myDatepicker></mat-datepicker>
-```
+An optional datepicker toggle button is available. 
```

Comment:
"There is also an optional datepicker toggle button that gives the user an easy way to open the datepicker pop-up"